x/market/internal/keeper/querier.go

```diff
@@ -14,8 +14,10 @@ func NewQuerier(keeper Keeper) sdk.Querier {
 		switch path[0] {
 		case types.QuerySwap:
 			return querySwap(ctx, req, keeper)
-		case types.QueryPrevDayIssuance:
-			return queryPrevDayIssuance(ctx, req, keeper)
+		case types.QueryPoolInfo:
+			return queryPoolInfo(ctx, req, keeper)
+		case types.QueryEqTerraPool:
+			return queryEqTerraPool(ctx, keeper)
```

Comment:
EqTerraPool is poorly named; consider making more informative

Reply:
Changed